### PR TITLE
HTTP API Specification

### PR DESCRIPTION
**Pull request by RichardLitt**
Working on the HTTP API Spec introduction blogpost

### CODE REVIEW

**Comment by RichardLitt** [comment]
This pertains to issue #63

File: drafts/http-api-spec/index.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+# Apiary
+
+![Screenshot of Apiary](apiary.png)
+
+We now have an HTTP API specification, available on the API-viewing website [Apiary](http://docs.ipfs.apiary.io/#). This means that you can browse the response for any IPFS command online.
+
+### Description of Apiary
+
+[Apiary](https://apiary.io) is an online service that provides a nice UI for looking at HTTP API specifications. It uses the [API Bluebprint](https://apiblueprint.org/), a simple and accessible language with concise and expressive syntax. The Apiary site reads from the [apiary.apib](https://github.com/ipfs/http-api-spec/blob/master/apiary.apib) file in the [ifps/http-api-spec](https://github.com/ipfs/http-api-spec) repository.
```
Comment (dignifiedquire) [inline drafts/http-api-spec/index.md]: `API Bluebprint` < typo

File: drafts/http-api-spec/skeleton.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Blog Post Skeleton
```
Comment (dignifiedquire) [inline drafts/http-api-spec/skeleton.md]: Why is this file here?

Comment (RichardLitt) [inline drafts/http-api-spec/skeleton.md]: Part of the process of building out a blog. Maybe shouldn't be added to the PR?

Comment (dignifiedquire) [inline drafts/http-api-spec/skeleton.md]: isn't it the wrong folder? (http-api-spec)

Comment (RichardLitt) [inline drafts/http-api-spec/skeleton.md]: No, we'll move it to the right one when it's ready to go.

**Comment by RichardLitt** [comment]
This should look a lot better now!

@whyrusleeping Can you help me out: what is a swarm address? How does that look?

**Comment by whyrusleeping** [comment]
A swarm address is a multiaddr

**Comment by diasdavid** [comment]
Announcing an HTTP-API spec that might not be totally coherent and complete doesn't sound like a good plan. The energy should be invested on https://github.com/ipfs/http-api-spec/issues/116 first, otherwise we will be pointing our users to something that won't work for them.

**Comment by RichardLitt** [comment]
That's fair. I think that the current HTTP API is useful, but I'm biased. 

@jbenet What do you think?

**Comment by diasdavid** [comment]
_Closing ancient posts that never got posted._
